### DISCUSSION TITLE
Can I set a shell command to execute only when a file is created in a particular folder?

### DISCUSSION THREAD

**Discussion by seatrout** [Taitava/obsidian-shellcommands#377] [Q&A]
I see there is an {{event_folder_name}} available in shell commands. Can I use this to ensure that a particular command is triggered only when a file is created in a particular folder?

(Use case: my markdownloader browser extension separates paragraphs automatically with two carriage returns. I have a script to change these to two spaces followed by one carriage return (the two spaces so they show up as breaks in reading view). I would like this script applied automatically to files moved into  my download directory.

So would something like

```
myscript -Params {{file_path:absolute}}  {{event_folder_name}}
if  {{event_folder_name}} == "Captures from Phone" {do stuff} ...
```
work?

**Comment by Taitava** [comment]
Hi! What shell are you using? Bash? PowerShell? Something else?

If it's Bash (or similar), you should be able to do something like:
```bash
if [ {{event_folder_name}} = "Captures from Phone" ]; then
    myscript -Params {{file_path:absolute}}  {{event_folder_name}}
fi
```

So, in other words, there's no setting in the SC plugin that would limit the file creation event to only execute when the file is created in a specific folder, but you can have the if condition in your shelll command, and if the condition is not met, your shell command still executes but does nothing.

Please let me know if this helped you, or if you have more questions. 🙂
